Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
 		stage('Check if UpdateStableIdentifiers build succeeded'){
 			steps{
 				script{
-					currentRelease = (pwd() =~ /Releases\/(\d+)\//)[0][1];
+					currentRelease = (pwd() =~/Releases\/(\d+)\//)[0][1];
```

Comment:
Just curious why the space after the tilde was removed here?

Reply:
That was a mistake, I've reverted it

Reply:
OMG really? it was a single space character... LOL ;)